### PR TITLE
Image & input components

### PR DESCRIPTION
**Pull request by kshitijthatte**
For image components, the following are added
- Responsive Images
- Square Images (available in 3 sizes)
- Round Images

For input components, the following are added
- Text Input with label
- Text Input with validation error

### CODE REVIEW

File: components/image/image.html [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+            or of a fixed size and shape.
+          </div>
+        </header>
+
+        <div class="divider"></div>
+
+        <div class="content">
+          <div class="content-title">Responsive Images</div>
+          <div class="description">
+            Images can bw made responsive with 'img-rsp' class. Use this if you
```
Comment (manojasarsa) [inline components/image/image.html]: A typo here - be*

**Review by manojasarsa — APPROVED** [review]
Rest is good to go! 👍

**Review by Sen-442b — APPROVED** [review]
Looks Good to Me

File: components/input/input.html [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+    &lt;input type="text" class="input" id="username" placeholder="Enter your username" /&gt;
+    &lt;label for="password"&gt;Password&lt;/label&gt;
+    &lt;input type="password" class="input" id="password" placeholder="Enter your password" /&gt;
+  &lt;/div&gt;
+                </code>
+            </pre>
+          </div>
+        </div>
+
+        <div class="divider"></div>
```
Comment (mimansha-swarup) [inline components/input/input.html:138]: you can use hr here

File: components/image/image.css [hunk 3]
```diff
@@ -0,0 +1,28 @@
+.img-rsp {
```
Comment (mimansha-swarup) [inline components/image/image.css]: i will suggest to keep class name more understandable like .img-responsive

**Review by mimansha-swarup — COMMENTED** [review]
few suggestions

**Review by MarkVed17 — APPROVED** [review]
Looks very clean @kshitijthatte !
You may proceed 👍🏼

**Comment by preethasri** [comment]
Looks 👍

File: components/image/image.html [hunk 4]
```diff
@@ -0,0 +1,198 @@
...
+    </nav>
+
+    <div class="container">
+      <aside class="drawer">
+        <div class="drawer-subheader">Get Started</div>
+        <ul>
+          <li class="drawer-item">
+            <a href="../../get-started/introduction.html">Introduction</a>
+          </li>
+          <li class="drawer-item"><a>Colors</a></li>
```
Comment (jainPrajwal) [inline components/image/image.html]: It's recommended to use the href attribute if you want to use an anchor tag for navigation purposes. It's semantically useful.

**Review by jainPrajwal — COMMENTED** [review]
Added a suggestion

File: components/image/image.html (hunk 1 above)
Comment (kshitijthatte) [reply components/image/image.html]: Thanks for pointing that out!

File: components/input/input.html (hunk 2 above)
Comment (kshitijthatte) [reply components/input/input.html:138]: Will have to fix that on every page, will do it.

File: components/image/image.css (hunk 3 above)
Comment (kshitijthatte) [reply components/image/image.css]: Thanks for the suggestion, done now!

File: components/image/image.html (hunk 4 above)
Comment (kshitijthatte) [reply components/image/image.html]: I thought I'll add it when I'll make that page, will add a placeholder href for now.

**Pull request merged**